EasyPost/Services/CarrierAccountService.cs

```diff
@@ -58,5 +66,25 @@ public async Task<CarrierAccount> Retrieve(string id)
         }
 
         #endregion
+
+        private static string GenerateEndpoint(string carrierAccountType)
```

Comment:
"Generate" probably isn't the right name for this since we aren't generating anything dynamic here. Maybe "Select" or "Route"?

Reply:
Resolved